calcom/cal.com
# Flicker on booking success page

## What does this PR do?

Fix the flicker on booking success page 
Loom video of the bug: https://www.loom.com/share/4d032cd2cb404618881fbdf4c6e0416a


Fixes #2614 [(issue)](https://github.com/calcom/cal.com/issues/2614)


Loom Video of the bug fixed: https://www.loom.com/share/eb7efdca32354701ba6436243be9ac69?t=0
## Type of change




## How should this be tested?

1. Go to Event Types  and click preview
2. When scheduling a meeting successfully the flicker will appear
